docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -36,3 +53,84 @@ function hs_paragraph_types_update_9401() {
     $paragraph->save();
   }
 }
+
+/**
+ * Update postcard body field text format.
+ */
+function hs_paragraph_types_update_9402() {
+  // Load filter_format and editor exported configuration.
+  $config_directory = new FileStorage(Settings::get('config_sync_directory'));
+  $filter_format_config = $config_directory->read('filter.format.minimal_html_with_styles');
```

Comment:
The service `config.storage.sync`. it does this part for you.
But this isn't needed anyways. The config will get created during the config import steps. This entire update hook is unnecessary

Reply:
@pookmish Fixed. We still need to check for and create the new filter format in order for the permissions to exist so we can grant them to the user roles, so that piece was moved to run in the same update hook